### PR TITLE
fix #7 table separate value accessor from label of column

### LINKED ISSUE #7: table: separate between label and column accessor in description
currently the name of the table column and the way how to access it within the raw data are coupled. To support longer names it should be decoupled.

### PR DESCRIPTION
**Pull request by sgratzl**
see #7.

### CODE REVIEW

File: src/table/Table.ts [hunk 1]
```diff
@@ -166,6 +168,7 @@ export function asTableFromArray(data: any[][], options: IAsTableOptions = {}):
   const columns = cols.map((col, i) => {
     return {
       name: col,
```
Comment (thinkh) [inline src/table/Table.ts:178]: Why do you keep `name` if it's equal to `column`?

Comment (sgratzl) [reply src/table/Table.ts:178]: what do you mean `name` is the label and `column` the accessor?

Comment (thinkh) [reply src/table/Table.ts:178]: See this line and the next line:
```typescript
name: col, // --> column object
column: col, // --> same column object 
```
I don't think that that `col` can be a string and an accessor at the same time.

Comment (thinkh) [reply src/table/Table.ts:178]: Same in line 192+193 (https://github.com/phovea/phovea_core/pull/83/files/b516b4dbbf10a879ff74c2fde45b9223bb10183c#diff-936ef1d331fb0454c0e0fececaa4e8edR192)

**Pull request merged**